src/main/resources/db/changelog/db-changelog-master.xml

```diff
@@ -0,0 +1,19 @@
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext
+   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd
+   http://www.liquibase.org/xml/ns/dbchangelog
+   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <include file="db/changelog/liquibase-outputChangeLog_init.xml"/>
+
+    <changeSet id="add new col" author="Wizzard">
+        <addColumn tableName="employee">
+            <column name="liquibaseColumn" type="VARCHAR(255)"/>
```

Comment:
pretty strange changeset, could you explain why do you add this column?

Reply:
need to show that i can add columns from liquibase, but i already have all the columns
is it better to remove the addition of an unnecessary column?

Reply:
Yes please. I certainly know you have good knowledge which allows you do a lot of manipulations with DB via Liquibase.

Reply:
I deleted it, thank you for your kind words)